doc/preferences.rst

```diff
@@ -0,0 +1,139 @@
+   Volume                            75.00          The percentage of the volume of the clip when added to the timeline
+   Blender Command (path)            *<blank>*      The path to the binary for Blender
+   Advanced Title Editor (path)      *<blank>*      The path to the binary for Inkscape
+   Show Export Dialog when Finished  *<checked>*    Displays the Export Video windows after the export is finished
+   ================================  =============  ===========
+
+Preview
+-------
+
+.. image:: images/preferences-2-preview.png
+
+Here you can set the Default Profile for your project, but you can always choose a different one when you start a new project. More about `profiles <profiles.html>`_.
+
+Here you can choose which connected audio device should be used by OpenShot. 
+If you experience sound issues, make sure that your audio settings match your audio card.
+The default audio sample rate of your sound device can be found under the sound device settings from your operating system. 
+
+.. table::
+   :widths: 30 20
+
+   ================================  ==================  ===========
+   Setting                           Default             Description
+   ================================  ==================  ===========
+   Default Video Profile             HD 720P 30 fps      Select the profile for Preview and Export defaults  
+   Playback Audio Device             Default             
+   Default Audio Sample Rate         44100               
+   Default Audio Channels            Stereo (2 Channel)  
+   ================================  ==================  ===========
+   
+.. TODO Add to FAQ, add Link [WIKI tag AUDIO + FAQ ?] here.  
+   For more info check our FAQ on the wiki.
+   Windows: disable audio enhancements/microphone recognition
+
+
+Autosave
+--------
+
+.. image:: images/preferences-3-autosave.png
+
+Autosave is a saving function in OpenShot which automatically saves the current changes or progress to your project, helping to reduce the risk or impact of data loss in case of a crash, freeze or user error.
+
+When loading projects from older versions of OpenShot, you may want to terporary disable Autosave to see if they work. This to prevent them from being automatically overwritten by a broken version. 
+.. Advise SuslikV on github https://github.com/OpenShot/openshot-qt/issues/3334
+
+Cache
+-----
+
+.. image:: images/preferences-4-cache.png
+
+A cache is a hardware or software component that stores data so that future requests for that data can be served faster.
+
+.. TODO Cache settings
+  CacheMode - When to use Disk? Servers?
+  Cache Limit (MB): How much % of RAM advised? Different for iGPU vs dedicated GPU?
+  Image Format: What is PPM/XMB/XPM? Where is this used for? 
+  Scale Factor: ??
+  Image Quality: ??
+
+Debug
+-----
+
+.. image:: images/preferences-5-debug.png
+
+Here you can modify how much data should be logged. Normally, *Debug Mode (verbose)* is off.
```

Comment:
Mention to only use "Verbose Debug Mode" when trying to troubleshoot a bug or crash, otherwise disable it. It slows down OpenShot quite a bit.